nss-copper-dragonflies/BangazonSite
# Ac productcategorieslist

# Description
Completed Ticket #1  and updated README
''' 
Given a user is viewing any page on the Bangazon site
When the user performs a gesture on the Product Categories hyperlink
Then the user will see a view containing a list of all product categories
And next to the category name, the number of products in that category will be displayed
And the first three products in the category will be displayed beneath each category, that are hyperlinks to the product detail

Sample
Electronics (15)
    Dryer
    HD Television
    Sony Walkman

Sporting Goods (7)
    Football
    Ice skates
    Basketball hoop
'''
## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. git fetch --all
1. git checkout ac-productcategorieslist 
1. ensure that when you can click on the product categories link even if not logged in
1. ensure that when you click on the product categories link in the nav bar, you are redirected to a view of all of the product categories available, a total product count per category and the first 3 products within the category. 

# Checklist:
